### ISSUE TITLE
help

### ISSUE THREAD

**Issue by treefreq** [Algolzw/image-restoration-sde#15]
During testing, my LQ data was 8X3X256X256, and the code could run. However, why is the dimension of the output data 1x3x256x256. Is there any way to make it output 8x3x256x256

**Comment by Algolzw** [comment]
Hi! Which model/task are you running? And in testing, we only support batch size to be "1" instead of "8".

**Comment by Algolzw** [comment]
If you really want to output the data more than one batch size, you can modify the [get_current_visuals](https://github.com/Algolzw/image-restoration-sde/blob/c00d4b49752de96b4735efe45ef5bd2f681629c4/codes/config/latent-dehazing/models/latent_denoising_model.py#LL196C24-L196C24) function:

change 
`out_dict["Output"] = self.output.detach()[0].float().cpu()` 
to 
`out_dict["Output"] = self.output.detach().float().cpu()`
